### PR TITLE
Resetting & snapshotting the blockchain - Closes #1288

### LINKED ISSUE #1288: Update how to reset the application
### Description
Describe `blockchain:reset` and what is does.

### URL
https://lisk.com/documentation/lisk-sdk/guides/app-development/setup.html#how-to-reset-the-database-of-an-application

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1288

### CODE REVIEW

**Review by mehmetegemen — DISMISSED** [review]
LGTM, good job Mona 🎉

File: modules/ROOT/pages/guides/node-management/blockchain-reset.adoc [hunk 1]
```diff
@@ -0,0 +1,103 @@
+= Resetting and snapshotting a node
+:toc:
+
+== Database locations
+
+The data of a blockchain application is stored in the following key-value stores:
+
+blockchain.db::
+Stores blocks, transactions and states.
```
Comment (chris529) [inline modules/ROOT/pages/guides/node-management/blockchain-reset.adoc:9]:
```suggestion
Stores blocks, transactions, and states.
```

File: modules/ROOT/pages/guides/node-management/blockchain-reset.adoc [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+All databases are located under `~/.lisk/BAPP-NAME/data/`
+
+If plugins are registered in the application, they will have a separate key-value store for each plugin for the plugin-specific off-chain data under `~/.lisk/BAPP-NAME/plugins/data/`.
+
+== Resetting the blockchain
+
+.This will remove all existing on-chain data of the blockchain application!
+[CAUTION]
+Meaning, the blockchain height will be back to 1 again, the next time the application the application is started and all existing accounts will be deleted, and the genesis accounts are reset to the state defined in the respective genesis block.
```
Comment (chris529) [inline modules/ROOT/pages/guides/node-management/blockchain-reset.adoc]:
```suggestion
Please note that the blockchain height will be reset back to 1 again the next time the application is started, and all existing accounts will be deleted. In addition, the genesis accounts will be reset to the state defined in the respective genesis block.
```

File: modules/ROOT/pages/guides/node-management/blockchain-reset.adoc [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+To reset the blockchain data of an application, it is therefore  only necessary to reset the `blockchain.db`.
+
+This can be achieved by using the `blockchain:reset` command of the application CLI.
+
+NOTE: Before using the command, make sure the application is stopped and not running anymore.
```
Comment (chris529) [inline modules/ROOT/pages/guides/node-management/blockchain-reset.adoc]:
```suggestion
NOTE: Before using the command, ensure the application is stopped and is not running anymore.
```

File: modules/ROOT/pages/guides/node-management/blockchain-reset.adoc [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+[source,bash]
+----
+bin/run blockchain:reset
+----
+
+== Creating snapshots of the blockchain
+
+To export the blockchain data of an application, run the following command:
+
+NOTE: Before using the command, make sure the application is stopped and not running anymore.
```
Comment (chris529) [inline modules/ROOT/pages/guides/node-management/blockchain-reset.adoc]:
```suggestion
NOTE: Before using the command, ensure the application is stopped and is not running anymore.
```

File: modules/ROOT/pages/guides/node-management/blockchain-reset.adoc [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+[source,bash]
+----
+% bin/run blockchain:export --help
+Export to <FILE>.
+
+USAGE
+  $ hello_app blockchain:export
+
+OPTIONS
+  -d, --data-path=data-path  Directory path to specify where node data is stored. Environment variable
```
Comment (chris529) [inline modules/ROOT/pages/guides/node-management/blockchain-reset.adoc]:
```suggestion
  -d, --data-path=data-path  Directory path to specify where the node data is stored. Environment variable
```

File: modules/ROOT/pages/guides/node-management/blockchain-reset.adoc [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+EXAMPLES
+  blockchain:export
+  blockchain:export --data-path ./data --output ./my/path/
+----
+
+== Importing a snapshot
+
+To import a snapshot of blockchain data, run the following command:
+
+NOTE: Before using the command, make sure the application is stopped and not running anymore.
```
Comment (chris529) [inline modules/ROOT/pages/guides/node-management/blockchain-reset.adoc]:
```suggestion
NOTE: Before using the command, ensure the application is stopped and is not running anymore.
```

File: modules/ROOT/pages/guides/node-management/blockchain-reset.adoc [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+Import from <FILE>.
+
+USAGE
+  $ hello_app blockchain:import FILEPATH
+
+ARGUMENTS
+  FILEPATH  Path to the gzipped blockchain data.
+
+OPTIONS
+  -d, --data-path=data-path  Directory path to specify where node data is stored. Environment variable
```
Comment (chris529) [inline modules/ROOT/pages/guides/node-management/blockchain-reset.adoc]:
```suggestion
  -d, --data-path=data-path  Directory path to specify where the node data is stored. Environment variable
```

**Review by chris529 — COMMENTED** [review]
Looks good, just a few minor points 👍

**Pull request merged**